algorithms/CPlusPlus/Trees/max-heap.cpp

```diff
@@ -0,0 +1,58 @@
+/*
+
+----Implementation of a Max Heap using priority queue from the STL library----
+
+List of member functions from the STL library for priority queues:
+-> empty() - returns whether the queue is empty
+-> size() - returns the size of the queue
+-> top() - access to the topmost element of the queue with a pointer
+-> push() - inserts element
+-> emplace() constructs and inserts element
+-> pop() - removes the top element
+-> swap()- swaps contents
+More information on this can be found at https://www.cplusplus.com/reference/queue/priority_queue/
+*/
+
+#include <iostream>
+#include <queue>
+#include <vector>
+
+using namespace std; //can also use std with the scope operator :: throughout the code or type "using namespace std::priority_queue" here
+
+void maxHeap() {
+    //creating the priority queue
+    //using a priority queue as is creates a max heap by default
+    std::priority_queue<int> pq;
+
+    // Push your values into the queue (the one's being used here are for testing purposes)
+    vector<int> values = {15,26,30,38,55,23,2,4,7,19,44,3,21,12,6}; //a vector of values of size 15
```

Comment:
```suggestion
void maxHeap(vector<int> &values) {
    //creating the priority queue
    //using a priority queue as is creates a max heap by default
    std::priority_queue<int> pq;
```

let's pass the list of values to the `maxHeap ()` and let the caller take care of creation of values that need to be heapified and give `maxHeap ()` the responsibility to only take care of creation of heap from those values.